x-pack/agent/docs/agent_configuration_example.yml

```diff
@@ -0,0 +1,209 @@
+outputs:
+  default:
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+    # Not supported at first
+    queue:
+      type: disk
+
+  long_term_storage:
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+    queue:
+      type: disk
+
+  monitoring:
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+
+settings.monitoring:
+  use_output: monitoring
+
+datasources:
+  # use the nginx package
+  - id: nginx-x1
+    title: "This is a nice title for human"
```

Comment:
I was leaning toward "title" because I would that using _description_ could be multiple lines where a title does not. but I do not have strong feeling to change it. Any opinion from the others.